src/com/projectkorra/projectkorra/waterbending/plant/PlantTether.java

```diff
@@ -0,0 +1,134 @@
+package com.projectkorra.projectkorra.waterbending.plant;
```

Comment:
This file seems to use spaces for indentation while the others use tabs

Reply:
fixed.